plugin/path_role.go

```diff
@@ -617,19 +596,6 @@ func (b *GcpAuthBackend) storeRole(ctx context.Context, s logical.Storage, roleN
 		return logical.ErrorResponse(err.Error()), nil
 	}
 
-	// Set default alias names
-	if role.IAMAliasType == "" {
-		role.IAMAliasType = defaultIAMAlias
-	}
-	if role.GCEAliasType == "" {
-		role.GCEAliasType = defaultGCEAlias
-	}
-
-	// Ensure a version is specified
-	if role.Version == 0 {
-		role.Version = currentGCPRoleVersion
```

Comment:
I think this renders `currentGCPRoleVersion` unused. Could remove it.

Reply:
Removed.